### DISCUSSION TITLE
Save screenshots directly to photos library on macOS

### DISCUSSION THREAD

**Discussion by Yushu2606** [ppy/osu#26663] [Ideas] [answered]
Maybe also on Android, iOS and iPadOS

**Comment by peppy** [comment, marked as answer]
On macOS this would be unwanted.

On mobile, use the screenshot key on your device for this.

**Reply by Yushu2606** [reply]
But if I want to use iCloud to synchronize screenshots, I still need to import them manually

**Reply by peppy** [reply]
Unfortunate but you'll have to get used to that workflow.

**Comment by Yushu2606** [comment]
And on iPadOS, if I use the shortcut key to take a screenshot, it will prompt that the screenshot has been saved, but I can not find osu! folder in File app

**Reply by peppy** [reply]
use the system keys. power + volume up.
